python/requirements/orca/requirement_base.txt

```diff
@@ -0,0 +1,11 @@
+# These are mirrored in python/orca/src/setup.py and python/dllib/src/setup.py 
+# as install_requires, which is what the users of the orca package will install.
+#
+# In short, if you change it here, PLEASE also change it in setup.py.
```

Comment:
shall we add this comment in each requirement?

Reply:
Maybe in base/automl and ray?

Reply:
Sure.